src/button.js

```diff
@@ -29,7 +32,7 @@ export class Button extends React.Component {
       return this.props.icon;
     }
 
-    return <i className={icon} />;
+    return (_.isString(icon) ? <i className={icon}/> : {icon});
```

Comment:
`icon` is a parameter that gets passed into `renderIcon`. If you look what gets passed in from the `render` function, it's always a string or empty string, which means we aren't checking the right variable here.